config.js

```diff
@@ -1,7 +1,7 @@
 let domain = 'meet.jit.si';
 
 module.exports = {
-    connection: {
+    connection: config || {
```

Comment:
@damencho, where should the config come from? Is it a global variable? I'm asking because I'm getting a 'ReferenceError: config is not defined' when I run 'npm run start:web' & I load the app.